### PR TITLE
Bump PHP 5.5 to 5.5.30

### PR DESCRIPTION
**Pull request by Zifius**

### CODE REVIEW

**Comment by ablyler** [comment]
@Zifius can you remove the bottle hashes for all formulas that start with php55 and contain `version PHP_VERSION`? Also, remove any `revision` lines from these formulas while you are at it.

**Comment by Zifius** [comment]
@ablyler sure, hopefully I did it right for both versions

File: Formula/php55.rb [hunk 1]
```diff
@@ -13,9 +13,6 @@ class Php55 < AbstractPhp
   head PHP_GITHUB_URL, :branch => PHP_BRANCH
```
Comment (ablyler) [inline Formula/php55.rb]: remove `revision 3` from above

File: Formula/php55-tidy.rb [hunk 2]
```diff
@@ -10,9 +10,6 @@ class Php55Tidy < AbstractPhp55Extension
   revision 2
```
Comment (ablyler) [inline Formula/php55-tidy.rb]: remove `revision 2` from above

File: Formula/php55.rb (hunk 1 above)
Comment (Zifius) [inline Formula/php55.rb]: All done

**Comment by ablyler** [comment]
@Zifius can you rebase against master to fix the merge conflict?

**Comment by Zifius** [comment]
@ablyler hope it's ok now, did `git fetch upstream` && `git rebase upstream/master` then resolved README.md conflict and then force pushed the updated branch

**Comment by ablyler** [comment]
@Zifius looks good, thanks!

**Comment by Zifius** [comment]
@ablyler happy to contribute, that hacktoberfest t-shirt won't qualify itself :) I missed the php56-typo revision removal, does it still makes sense to fix now or with a next PHP release?

**Pull request closed without merging**